eqt/ui/UIFormWidget.py

```diff
@@ -122,6 +122,19 @@ def __init__(self, parent=None, title=None):
     def addWidget(self, qwidget, qlabel, name):
         self.widget().addWidget(qwidget, qlabel, name)
 
+    def getWidget(self, name, role='widget'):
+        '''returns the Widget by the name with which it has been added
+        
+        By default it returns the widget that is the field in the form. 
+        The user can get the label by specifying the role to be label
+        
+        Raises ValueError if the role is not widget or label.
+        '''
+        allowed_roles = ['widget', 'label']
```

Comment:
This should be 'field' not 'widget', see the _addWidget method in UIFormWidget above where the names get set

Reply:
Make sure you test it works for both widgets and labels!